reference/nats-protocol/nats-protocol/README.md

```diff
@@ -119,20 +134,26 @@ The `CONNECT` message is the client version of the [`INFO`](./#info) message. On
+| `auth_token`    | Client authorization token.                                                                                                                                                                                                                                                       | string | if `auth_required` is `true` |
+| `user`          | Connection username.                                                                                                                                                                                                                                                              | string | if `auth_required` is `true` |
+| `pass`          | Connection password.                                                                                                                                                                                                                                                              | string | if `auth_required` is `true` |
+| `name`          | Client name.                                                                                                                                                                                                                                                                      | string | false                        |
+| `lang`          | The implementation language of the client.                                                                                                                                                                                                                                        | string | true                         |
+| `version`       | The version of the client.                                                                                                                                                                                                                                                        | string | true                         |
+| `protocol`      | Sending `0` (or absent) indicates client supports original protocol. Sending `1` indicates that the client supports dynamic reconfiguration of cluster topology changes by asynchronously receiving [`INFO`](./#info) messages with known servers it can reconnect to.            | int    | false                        |
+| `echo`          | If set to `false`, the server (version 1.2.0+) will not send originating messages from this connection to its own subscriptions. Clients should set this to `false` only for server supporting this feature, which is when `proto` in the `INFO` protocol is set to at least `1`. | bool   | false                        |
```

Comment:
to note - I have flipped the description on this from `true` to `false` since I think it was documented incorrectly. Happy to discuss if necessary.

Reply:
You are correct, default opts on the server show it is set to `true`.